src/libsync/syncjournaldb.cpp

```diff
@@ -80,12 +84,36 @@ bool SyncJournalDb::maybeMigrateDb(const QString& localPath, const QString& abso
             return false;
         }
     }
+    if( FileSystem::fileExists( newDbNameWal ) ) {
+        if( !FileSystem::remove(newDbNameWal, &error) ) {
+            qDebug() << "Database migration: Could not remove db WAL file" << newDbNameWal
+                     << "due to" << error;
+            return false;
```

Comment:
i think we could ignore this error and not return false.  (everywhere on these new checks)

Because this is not a fatal error, is it?